modules/styles.py

```diff
@@ -8,6 +8,8 @@
 import collections.abc as abc
 import tempfile
 import shutil
+from _csv import register_dialect, QUOTE_MINIMAL
+from csv import Dialect

```

Comment:
```suggestion
from csv import excel, register_dialect
```

Reply:
thank you.
I am not proficient in Python's OOM/custom, above is just a way to solve the errors by copying the source code